test/models/thesis_test.rb

```diff
@@ -598,19 +598,20 @@ def attach_file_with_purpose_to(thesis, purpose = 'thesis_pdf')
     thesis.save
     thesis.reload
     # Fixture meets all conditions
-    assert_equal true, thesis.valid?
-    assert_equal true, thesis.files?
-    assert_equal true, thesis.files_have_purpose?
-    assert_equal true, thesis.files_complete
-    assert_equal true, thesis.metadata_complete
-    assert_equal false, thesis.issues_found
-    assert_equal true, thesis.no_issues_found?
-    assert_equal true, thesis.authors_graduated?
-    assert_equal false, thesis.active_holds?
-    assert_equal true, thesis.no_active_holds?
-    assert_equal true, thesis.departments_have_dspace_name?
-    assert_equal true, thesis.degrees_have_types?
-    assert_equal true, thesis.accession_number.present?
+    assert thesis.valid?
```

Comment:
These changes are just to use the preferred syntax for asserting truthy/falsey:

https://minitest.rubystyle.guide/#assert-truthy